exchangelib/folders.py

```diff
@@ -117,6 +117,12 @@ def __init__(self, id, changekey):
         self.id = id
         self.changekey = changekey
 
+    @property
+    def item_id(self):
```

Comment:
You are entirely correct, and I'm mistaken. `ItemId` isn't supported as input. Probably it should be, but I agree it's out of scope to fix it here.